flytekit/clis/sdk_in_container/package.py

```diff
@@ -40,7 +41,17 @@
     required=False,
     type=click.Path(exists=True, file_okay=False, readable=True, resolve_path=True, allow_dash=True),
     default=".",
-    help="Local filesystem path to the root of the package.",
+    help="Local filesystem path to the root of the package. Example: --source /path/to/workflows",
+)
+@click.option(
```

Comment:
It is not correct to add a `"--pkgs"` option here, this is an option of the parent command `pyflyte` itself, not for the `package` sub command. Let's remove this again in favour of what I propose below.

---

> Please note that this "
    "option will override the option specified in the configuration file, or environment variable"

Could you please point me to which env var and config arg control this, haven't used them before?

---

The problem is that assumably most users will search in `pyflyte package --help` instead of `pyflyte --help` so it won't be very helpful to put the hint in the `--pkgs` option in the parent command.

Instead, I'd propose to add something like this to the [main help string](https://github.com/flyteorg/flytekit/blob/ff2d0da686c82266db4dbf764a009896cf062349/flytekit/clis/sdk_in_container/package.py#L128) of `pyflyte package`:
 
Given a python package containing flyte entities with e.g. such a structure ...

```
.
└── some_package
    ├── __init__.py
    └── some_module
        ├── __init__.py
        └── wf.py
```

... execute `pyflyte --pkgs some_package package` to package the flyte entities contained in the package.



This hopefully makes it very clear in the first location where people will look ...